dist/angular-foundation-no-tpls.js

```diff
@@ -287,63 +288,72 @@ angular.module('mm.foundation.dropdownMenu', []).directive('dropdownMenu', ['$co
             'ngInject';
 
             var vm = this;
-            // $element is-dropdown-submenu-parent
         }]
     };
-}]).directive('li', function () {
-    return {
+}]).directive('li', ['$timeout', dropdownMenuItem]);
+
+dropdownMenuItem.$inject = ['$timeout'];
```

Comment:
We're using angular-annotate so there is no need for setting $inject manually. Could you instead add `'ngInject';` on the first line of the body of `dropdownMenuItem`?